.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,161 @@
+          export PREFIX=/tmp/$USER/$GITHUB_SHA
+          export PATH=$PREFIX/bin:$(cat $HOME/path)
+          export PYTHONPATH=$PREFIX/lib/python${{matrix.python}}/site-packages:$PYTHONPATH
+          eb --version
+          # tell EasyBuild which modules tool is available
+          if [[ ${{matrix.modules_tool}} =~ ^modules-tcl- ]]; then
+            export EASYBUILD_MODULES_TOOL=EnvironmentModulesTcl
+          elif [[ ${{matrix.modules_tool}} =~ ^modules-3 ]]; then
+            export EASYBUILD_MODULES_TOOL=EnvironmentModulesC
+          elif [[ ${{matrix.modules_tool}} =~ ^modules-4 ]]; then
+            export EASYBUILD_MODULES_TOOL=EnvironmentModules
+          else
+            export EASYBUILD_MODULES_TOOL=Lmod
+          fi
+          export TEST_EASYBUILD_MODULES_TOOL=$EASYBUILD_MODULES_TOOL
+          eb --show-config
+          # gather some useful info on test system
+          eb --show-system-info
+          # check GitHub configuration
+          eb --check-github --github-user=easybuild_test
+          # run test suite
+          python -O -m test.framework.suite 2>&1 | tee test_framework_suite.log
+          # try and make sure output of running tests is clean (no printed messages/warnings)
+          IGNORE_PATTERNS="no GitHub token available|skipping SvnRepository test|requires Lmod as modules tool|stty: 'standard input': Inappropriate ioctl for device"
+          # '|| true' is needed to avoid that Travis stops the job on non-zero exit of grep (i.e. when there are no matches)
+          PRINTED_MSG=$(egrep -v "${IGNORE_PATTERNS}" test_framework_suite.log | grep '\.\n*[A-Za-z]' || true)
+          test "x$PRINTED_MSG" = "x" || (echo "ERROR: Found printed messages in output of test suite\n${PRINTED_MSG}" && exit 1)
+
+          # version and SHA256 checksum are hardcoded below to avoid forgetting to update the version in the script along with contents
+          EB_BOOTSTRAP_VERSION=$(grep '^EB_BOOTSTRAP_VERSION' easybuild/scripts/bootstrap_eb.py | sed 's/[^0-9.]//g')
+          EB_BOOTSTRAP_SHA256SUM=$(sha256sum easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
+          EB_BOOTSTRAP_FOUND="$EB_BOOTSTRAP_VERSION $EB_BOOTSTRAP_SHA256SUM"
+          EB_BOOTSTRAP_EXPECTED="20190922.01 778762479d77df641c247aba33c13203c8c5b8275e47f59464bab93917aaaf22"
+          test "$EB_BOOTSTRAP_FOUND" = "$EB_BOOTSTRAP_EXPECTED" || (echo "Version check on bootstrap script failed $EB_BOOTSTRAP_FOUND" && exit 1)
+
+          # test bootstrap script (only compatible with Python 2 for now)
+          if [[ ${{matrix.python}} =~ '2.' ]]; then
+              export PREFIX=/tmp/$USER/$GITHUB_SHA/eb_bootstrap
+              python easybuild/scripts/bootstrap_eb.py $PREFIX
+              # unset $PYTHONPATH to avoid mixing two EasyBuild 'installations' when testing bootstrapped EasyBuild module
+              unset PYTHONPATH
+              # simply sanity check on bootstrapped EasyBuild module (skip when testing with Python 3, for now)
```

Comment:
Do you mean "simple sanity check on" or "simply sanity check the ..."

Reply:
`simple`, will fix